src/libraries/System.Private.CoreLib/src/System/String.cs

```diff
@@ -31,8 +31,9 @@ public sealed partial class String : IComparable, IEnumerable, IConvertible, IEn
         [NonSerialized]
         private readonly int _stringLength;
 
-        // For empty strings, this will be '\0' since
-        // strings are both null-terminated and length prefixed
+        // For empty strings, this will be '\0' since strings are both null-terminated and length prefixed.
+        // Also readonly, however String uses .ctors that C# doesn't recognise as .ctors;
+        // so it complains about modifitcation outside a constructor.
```

Comment:
```suggestion
        // so it complains about modification outside a constructor.
```

Reply:
I need to install a spell checker extension :)